.github/workflows/milestone.yml

```diff
@@ -0,0 +1,37 @@
+name: milestone
+
+on:
+  pull_request:
+    branches: [master]
+    types: [closed]
+
+jobs:
+  set-milestone:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Get milestone from pom.xml
+        run: |
+          echo "MILESTONE_NUMBER=$(./mvnw -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec | cut -d- -f1)" >> $GITHUB_ENV
```

Comment:
Nit: could shorten `--non-recursive` to `-N`

We do this a different way in a couple other places:
```
./mvnw -q help:evaluate -Dexpression=project.version -DforceStdout
```

But the `exec:exec` way is about **1.0s** vs **1.6s** for `help:evaluate` on my machine, so we could switch everywhere.

Reply:
Exactly, `exec:exec` is faster than `help:evaluate` on my machine too. Let me handle in follow-up PR.